plugin/pkg/admission/resourcequota/controller.go

```diff
@@ -382,6 +402,14 @@ func (e *quotaEvaluator) checkRequest(quotas []api.ResourceQuota, a admission.At
 	// if we have limited resources enabled for this resource, always calculate usage
 	inputObject := a.GetObject()
 
+	// Check if object matches AdmissionConfiguration matchScopes
+	limitedScopes, err := getMatchedLimitedScopes(evaluator, inputObject, e.config.LimitedResources)
```

Comment:
Why the MatchScopes and MatchContains are managed separately?
I think they can be managed together.

Reply:
Since Scopes and resources are conceptually different and required different logic at evaluators as well, it is prefered to keep seperate APIs for each. Both may be managed together but that might make implementation complex.

Reply:
OK :+1: